<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication2Class</class>
 <widget class="QMainWindow" name="QtGuiApplication2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtGuiApplication2.qrc">
    <normaloff>:/QtGuiApplication2/Resources/icon.png</normaloff>:/QtGuiApplication2/Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="actionSimple_calculate"/>
    <addaction name="actionGradepoint_calculate"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout_Dev"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actionSimple_calculator_config"/>
   </widget>
   <addaction name="menuFunction"/>
   <addaction name="menuSetting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSimple_calculate">
   <property name="text">
    <string>simple calculate</string>
   </property>
  </action>
  <action name="actionGradepoint_calculate">
   <property name="text">
    <string>gradepoint calculate</string>
   </property>
  </action>
  <action name="actionAbout_Dev">
   <property name="icon">
    <iconset resource="QtGuiApplication2.qrc">
     <normaloff>:/QtGuiApplication2/Resources/about.png</normaloff>:/QtGuiApplication2/Resources/about.png</iconset>
   </property>
   <property name="text">
    <string>About this app</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="actionSimple_calculator_config">
   <property name="text">
    <string>simple calculator config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication2.qrc"/>
 </resources>
 <connections/>
</ui>
